src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -276,6 +278,10 @@ class Microsoft::Terminal::Core::Terminal final :
     size_t _hyperlinkPatternId;
 
     std::wstring _workingDirectory;
+
+    // This font value is only used to check if the font is a raster font.
+    // Otherwise, the font is changed with the renderer via TriggerFontChange.

```

Comment:
This comment is no longer true. UiaTextRangeBase now uses the actual font name from this

Reply:
Nah, it's still true. Maybe I just need to word smith it a bit haha.

What I'm trying to say is that we're initializing `_fontInfo` to a fake value that we don't actually intend on using. The _only_ thing we'll use this fake value for is to check if it's a raster font.